about.html

```diff
@@ -0,0 +1,136 @@
+          /></a>
+          <a href="about.html">About</a>
+          <a href="#media">Media</a>
+          <a href="#lineup">Line up</a>
+          <a href="#buytickets">Buy tickets</a>
+          <div class="menu-close">
+            <img src="assets/img/Union.svg" alt="" />
+          </div>
+        </div>
+      </nav>
+    </section>
+    <header class="about about-bg">
+      <p class="hello about">"Hello Micronauts!"</p>
+      <h1 class="title about">Microverse Concert Music for Developers 2021</h1>
+      <div class="intro about">
+        <p>
+          Lorem ipsum dolor sit amet consectetur, adipisicing elit. Officiis
+          nulla animi praesentium cumque hic! Est quis dolorum quidem itaque,
+          voluptatum impedit expedita. Minus itaque nisi magnam perferendis
+          quas? <br />
+          Incidunt provident rem accusamus adipisci, quisquam perspiciatis
+          temporibus! Rerum sapiente dolores possimus unde odio nostrum minima,
+          inventore reprehenderit commodi debitis perspiciatis saepe aliquam
+          impedit molestiae culpa dignissimos reiciendis in soluta! <br />
+          Apestius dolor sit, amet consectetur adipisicing elit. Recusandae,
+          sunt.
+        </p>
+        <p>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Accus antium
+          ipsam iste itaque nihil architecto, volu tatem sed consectetur
+          adipisicing elit facilis porro. Inventore necessitatibus suscipit
+          distinctio quidem dolorem placeat! Ad autem nisi vel repellendus.
+          Iste!
+        </p>
+      </div>
+      <p class="at about">
+        Please contact us per Email for any further questions about Microverse
+        Concert Music for Developers 2021! <br /><strong
+          >concert@microverse.org</strong
+        >
+      </p>
+    </header>
+    <section class="featured about">
+      <h2>Microverse Concert Music for Developers 2021 Logo</h2>
+      <p class="subtitle about">
+        The logo of Microverse Concert Music for Developers was created by his
+        founder in 2015.
+      </p>
+      <p class="subtitle full about">
+        Lorem ipsum dolor sit amet consectetur, adipisicing elit. Quia minima
+        voluptatem dolor unde tenetur expedita distinctio! Ab provident,
+        accusantium at sequi possimus ut! Molestiae earum necessitatibus
+        deserunt corrupti quasi blanditiis! Autem praesentium consequuntur
+        nesciunt? Consequuntur alias rerum laborum harum! Placeat?
+      </p>
+      <div class="logo about">
+        <span class="aligner"></span>
+        <img src="assets/img/logo.png" alt="" />
+      </div>
+      <div class="separator"></div>
+      <h2>See the past Microverse Concerts</h2>
+      <p class="subtitle about">
+        Take a look at the last two Microverse Concert in San Francisco and New
+        York.
+      </p>
+      <div class="concerts-wrap about">
+        <div class="concert1">
```

Comment:
- [ ] Kindly check out my comments about semantic HTML tags above and how to use them.
- [ ] The header intro should be a dark grey and should a larger and bolder font. The same goes for the contact info, and the middle section(about the concert section).

Reply:
Thank you @temmarie 🙌